src/main/java/io/appform/dropwizard/actors/common/Constants.java

```diff
@@ -6,8 +6,9 @@
 @NoArgsConstructor(access = AccessLevel.PRIVATE)
 public class Constants {
 
-    public static final String DEFAULT_CONNECTION_NAME = "default";
+    public static final String DEFAULT_PRODUCER_CONNECTION_NAME = "defaultproducer";
```

Comment:
We have two default connections now. There is a connection registry which has list of custom connections which application can create. Ideally we should prevent users from mentioning these two connection names in the other list since the threadpool config used would be the default config rather than that connection specific config

Reply:
Makes sense, added a check for it